packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -27,8 +27,11 @@ export class TdsTooltip {
   /** Allow mouse over Tooltip. Useful when Tooltip contains clickable elements like link or button. */
   @Prop() mouseOverTooltip: boolean = false;
 
+  /** What triggers the popover to show */
+  @Prop() trigger: 'click' | 'hover' = 'hover';
+
   /** Prop in control of showing and hiding prop */
-  @Prop({ mutable: true }) show: boolean = false;
+  @Prop({ mutable: true }) show: boolean = null;
```

Comment:
```suggestion
  @Prop({ mutable: true }) show: boolean = false;
```
As prop is defined as a boolean, I propose we keep using false here unless there is a specific reason to keep it null.

Reply:
I agree, although when i change all the flags to false `popover-canvas` & `popover-menu` are effected since we rely on default value to be null when flag is not provided at all.
My suggestion is to merge this or to totally revisit testing and logic around show flags in all components using `popover-core`.

Note: I agree, but it works for now :)

Reply:
I agree with @Lunkan89, we need to keep it undefined/null if the users hasn't used it. Need to make sure to differentiate between controlled vs uncontrolled state of components. Same should go for modal/popover/tooltip

Reply:
Then all good, I missed the part on dependency in popover-canvas and popover-menu.